modules/artifact/src/main/java/org/mule/module/artifact/classloader/JMSResourceReleaser.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.artifact.classloader;
+
+import static org.mule.module.artifact.classloader.ThreadCommonMethodsUtil.*;
+import static java.lang.Thread.activeCount;
+import static java.lang.Thread.enumerate;
+import static org.slf4j.LoggerFactory.getLogger;
+
+import org.mule.runtime.module.artifact.api.classloader.ArtifactClassLoader;
+import org.mule.runtime.module.artifact.api.classloader.ResourceReleaser;
+
+import org.slf4j.Logger;
+
+/**
+ * Class created to solve classloading leaks
+ */
+public class JMSResourceReleaser implements ResourceReleaser {
```

Comment:
If the target is only activeMQ please rename the class because of this. If more releasers associated to Activemq are added in the future, a composite releaser can be made. But apparently this only applies to ActiveMQ